src/box/errcode.h

```diff
@@ -115,286 +115,286 @@ struct errcode_record {
-	_(ER_MISSING_REQUEST_FIELD, 69,		"Missing mandatory field '%s' in request") \
-	_(ER_IDENTIFIER, 70,			"Invalid identifier '%s' (expected printable symbols only or it is too long)") \
-	_(ER_DROP_FUNCTION, 71,			"Can't drop function %u: %s") \
-	_(ER_ITERATOR_TYPE, 72,			"Unknown iterator type '%s'") \
-	_(ER_REPLICA_MAX, 73,			"Replica count limit reached: %u") \
-	_(ER_INVALID_XLOG, 74,			"Failed to read xlog: %lld") \
-	_(ER_INVALID_XLOG_NAME, 75,		"Invalid xlog name: expected %lld got %lld") \
-	_(ER_INVALID_XLOG_ORDER, 76,		"Invalid xlog order: %lld and %lld") \
+	_(ER_INSTANCE_UUID_MISMATCH, 66,	"Instance UUID mismatch", "expected", STRING, "actual", STRING) \
+	_(ER_REPLICA_ID_IS_RESERVED, 67,	"Can't initialize replica id with a reserved value", "reserved_id", UINT) \
+	/* ER_INVALID_ORDER, 68, Unused */ \
+	_(ER_MISSING_REQUEST_FIELD, 69,		"Missing mandatory field in request", "field", STRING) \
+	_(ER_IDENTIFIER, 70,			"Invalid identifier", "id", STRING, "details", STRING) \
+	_(ER_DROP_FUNCTION, 71,			"Can't drop function", "function_id", UINT, "details", STRING) \
+	_(ER_ITERATOR_TYPE, 72,			"Unknown iterator type", "type", STRING) \
+	_(ER_REPLICA_MAX, 73,			"Replica count limit reached", "limit", UINT) \
+	/* ER_INVALID_XLOG, 74, Unused */ \
+	/* ER_INVALID_XLOG_NAME, 75, Unused */ \
+	/* ER_INVALID_XLOG_ORDER, 76, Unused */ \
 	_(ER_NO_CONNECTION, 77,			"Connection is not established") \
 	_(ER_TIMEOUT, 78,			"Timeout exceeded") \
 	_(ER_ACTIVE_TRANSACTION, 79,		"Operation is not permitted when there is an active transaction ") \
 	_(ER_CURSOR_NO_TRANSACTION, 80,		"The transaction the cursor belongs to has ended") \
 	_(ER_CROSS_ENGINE_TRANSACTION, 81,	"A multi-statement transaction can not use multiple storage engines") \
-	_(ER_NO_SUCH_ROLE, 82,			"Role '%s' is not found") \
-	_(ER_ROLE_EXISTS, 83,			"Role '%s' already exists") \
-	_(ER_CREATE_ROLE, 84,			"Failed to create role '%s': %s") \
-	_(ER_INDEX_EXISTS, 85,			"Index '%s' already exists") \
+	_(ER_NO_SUCH_ROLE, 82,			"Role does not exist", "role", STRING) \
+	_(ER_ROLE_EXISTS, 83,			"Role already exists", "role", STRING) \
+	_(ER_CREATE_ROLE, 84,			"Failed to create role", "role", STRING, "details", STRING) \
+	_(ER_INDEX_EXISTS, 85,			"Index already exists", "index", STRING) \
 	_(ER_SESSION_CLOSED, 86,		"Session is closed") \
-	_(ER_ROLE_LOOP, 87,			"Granting role '%s' to role '%s' would create a loop") \
-	_(ER_GRANT, 88,				"Incorrect grant arguments: %s") \
-	_(ER_PRIV_GRANTED, 89,			"User '%s' already has %s access on %s%s") \
-	_(ER_ROLE_GRANTED, 90,			"User '%s' already has role '%s'") \
-	_(ER_PRIV_NOT_GRANTED, 91,		"User '%s' does not have %s access on %s '%s'") \
-	_(ER_ROLE_NOT_GRANTED, 92,		"User '%s' does not have role '%s'") \
+	_(ER_ROLE_LOOP, 87,			"Granting role would create a loop", "role", STRING, "grantee", STRING) \
+	_(ER_GRANT, 88,				"Granting failed", "details", STRING) \
```

Comment:
"Cannot update privilege" or "Failed to grant privilege"

Reply:
Fixed.

```diff
+       _(ER_GRANT, 88,                         "Cannot update privilege", "details", STRING) \
```